x-pack/build.gradle

```diff
@@ -0,0 +1,29 @@
+description = """Logstash X-Pack"""
```

Comment:
Since we're adding this file, we need to set `isDistributedArtifact=true` as we do in `logstash-core` here https://github.com/elastic/logstash/blob/master/logstash-core/gradle.properties . So we need to add a `gradle.properties `for this subproject

Reply:
@andrewvc but it isn't a distributed artefact is it? We're not packaging any jar here at this point?

Reply:
Not now, but, if at some point we start adding Java code, and I'm sure we will, we'll probably forget to add that variable. We should add that now along with the build.gradle so we don't get into a sticky situation later.

So, you're right, it's a bit of a noop now, but I think it's the right thing to do.

We can also just merge this now and open a new PR if you prefer. I'm glad to do it. Your choice.